LABS.md

```diff
@@ -53,7 +53,7 @@ download: git-scm.com, or use your favourite package manager
 1. compare the directory structure with your original repository
 1. switch to the terminal with your original repository
 1. add the new bare repository as remote `git remote add origin ../demo-bare.git`
-1. push your current branch into your remote named `origin`
+1. push your current branch into your remote named `origin` 'git push origin master'
```

Comment:
this is now off compared to the rest, please replace the `origin` part with your addition but make sure to use backticks instead of quotes because that looks more pretty on the markdown preview.

Reply:
updated, new pull request will be issued